src/Tribe/Views/V2/View.php

```diff
@@ -942,12 +956,18 @@ protected function setup_repository_args( \Tribe__Context $context = null ) {
 
 		$context_arr = $context->to_array();
 
-		return [
+		$args =  [
 			'posts_per_page' => $context_arr['events_per_page'],
 			'paged' => max( Arr::get_first_set( array_filter( $context_arr ), [ 'paged', 'page' ], 1 ), 1 ),
 			'search' => $context->get( 'keyword', '' ),
 			'hidden_from_upcoming' => false,
 		];
+
+		if( $this->should_reset_page() ){
```

Comment:
* :no_entry_sign: There must be a space between if and the first parenthesis '('
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after IF keyword; 0 found
* :no_entry_sign: You must have a space after a closing parenthesis
* :no_entry_sign: Expected 1 space(s) after closing parenthesis; found 0
* :no_entry_sign: Space between opening control structure and closing parenthesis is required

<sub>hash: 2624b68da1864fd07c22135cb3af8a4c</sub>